content/docs/runbooks/admin/remove-kudo.md

```diff
@@ -0,0 +1,32 @@
+# How to uninstall KUDO
+
+The objective of this runbook is to completely uninstall KUDO from a kubernetes cluster.
+
+## Preconditions
+
+KUDO is initialized and running on a cluster.
+
+## Steps
+
+### Uninstall KUDO
+
+`kubectl kudo init --dry-run --output yaml | kubectl delete -f-`
+
```

Comment:
Should we add a warning that this removes all installed operators and deployments as well?